sopel/tools/jobs.py

```diff
@@ -20,20 +20,14 @@
     import queue as Queue
 
 
-class released(object):
```

Comment:
I would make noise about marking this as deprecated, but the [changelog for 5.3.0](https://sopel.chat/changelog/5.3.0/) makes this easy:

> * Functions and classes used internally for scheduling jobs are moved. They are still available in their old location, but are no longer documented and will not be considered part of the public API in 6.0.

Since this is part of the job tools, it's not public, so we don't have to go through a deprecation cycle. :tada:

Reply:
Oh, that's interesting! I'll add a comment about that in the code, so future devs won't be surprised it's not included in the public documentation.